assets/stylesheets/modules/_buttons.scss

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   Buttons
+   ========================================================================== */
+
+/**
+ * Inherits from pasteup buttons
+ */
+@import '../bower_components/pasteup-buttons/src/_buttons';
+
+.button--large {
+    cursor: pointer;
+    @include font-size(14px, 34px);
+    min-width: 160px;
```

Comment:
Yep that's what this PR covers https://github.com/guardian/subscriptions-frontend/pull/76 , no need to wrap anything in `rem()` anymore.